node/node.go

```diff
@@ -962,6 +962,10 @@ func (n *Node) GetAccountWithKeys(address string, options api.AccountQueryOption
 
 	var keys map[string]string
 	if options.WithKeys {
+		if accInfo.account == nil || accInfo.account.DataTrie() == nil {
+			return accInfo.accountResponse, accInfo.block, nil
```

Comment:
Shouldn't we return the default values like in other cases?
``` go
return api.AccountResponse{}, api.BlockInfo{}, err
```

Reply:
the accountResponse and BlockInfo come populated even if the accounts are empty or have nil datatries. So you would get a valid accountResponse with nonce, address etc and a valid blockInfo even in the case of nil account or datatrie.